contrib/python/tests/python/pants_test/contrib/python/checks/checker/test_print_statements.py

```diff
@@ -29,8 +28,8 @@ def test_print_function(self):
     """
     self.assertNoNits(statement)
 
-  @unittest.skipIf(PY3, reason='Print statement check disabled on Python 3 because interpreter '
-                               'will already throw a syntax error.')
+  # TODO(#7979): Rework tests so that we can run this with Python 2.
+  @unittest.skip
```

Comment:
Could we remove this test altogether? If the users _can_ run pants under python2, we should still only `skipif(PY3`. If the user _can't_ run this under Python 2, there's not much sense keeping this around.

Reply:
No, we should not remove it. This code under `contrib.python.checks.checker` is special in that it may be ran under Py2 or Py3, depending on what `select-interpreter` chooses.